app/controllers/user_stories_controller.rb

```diff
@@ -62,6 +62,9 @@ def update
       render 'edit' and return false
     end
     redirect_back_or(project_backlog_index_path(@project))
+  rescue ActiveRecord::StaleObjectError
+    flash.now[:error] = "Another person has just updated that record"
+    render 'edit' and return false
```

Comment:
Use `&&` instead of `and`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.